manifests/a/AxisCommunications/NetEye200/latest/AxisCommunications.NetEye200.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: AxisCommunications.NetEye200
+PackageVersion: latest

```

Comment:
```suggestion
PackageVersion: 1.32
```
`latest` will now allow winget to understand if the package can be upgraded or not. Please change this to a semantic version. 

![image](https://github.com/microsoft/winget-pkgs/assets/12611259/ce687628-0cd3-407f-9c38-8628740b783b)